src/hooks/useCart/useCart.tsx

```diff
@@ -138,11 +143,16 @@ export const useCart = (
       setCartState("FETCHING_QUOTA");
       try {
         const quotaResponse = await getQuota(ids, authKey, endpoint);
-        if (hasNoQuota(quotaResponse)) {
+        if (hasInvalidQuota(quotaResponse)) {
+          Sentry.captureException(
+            `Negative Quota Received: ${JSON.stringify(
+              quotaResponse.remainingQuota
+            )}`
+          );
           setCartState("NO_QUOTA");
-        } else {
-          setCartState("DEFAULT");
-        }
+        } else if (hasNoQuota(quotaResponse)) {
+          setCartState("NO_QUOTA");
+        } else setCartState("DEFAULT");
```

Comment:
Minor nitpick, it's more readable if we follow the convention of having all conditions in curly brackets
```suggestion
        } else {
          setCartState("DEFAULT");
        }
```